docs/userguides/testing.md

```diff
@@ -35,6 +35,9 @@ def test_is_owner(my_contract, owner, other):
     assert not other_is_owner
 ```
 
+```{note} ape has built-in test and fixture isolation for all pytest scopes. To disable isolation add the `--disable-isolation` flag when running `ape test`
```

Comment:
```suggestion
```{note}
Ape has built-in test and fixture isolation for all pytest scopes.
To disable isolation add the `--disable-isolation` flag when running `ape test`
```